packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -61,10 +61,15 @@ export default function Preview( { idBase, instance, isVisible } ) {
 			function setHeight() {
 				// Pick the maximum of these two values to account for margin collapsing.
 				const height = Math.max(
-					iframe.contentDocument.documentElement.offsetHeight,
-					iframe.contentDocument.body.offsetHeight
+					iframe.contentDocument.documentElement?.offsetHeight ?? 0,
+					iframe.contentDocument.body?.offsetHeight ?? 0
 				);
-				iframe.style.height = `${ height }px`;
+
+				// Fallback to a height of 100px if the height cannot be determined.
+				// This ensures the block is still selectable. 100px should hopefully
+				// be not so big that it's annoying, and not so small that nothing
+				// can be seen.
+				iframe.style.height = `${ height !== 0 ? height : 100 }px`;
```

Comment:
A nit but `Math.max` supports three arguments so this could be:

```js
const height = Math.max(
	iframe.contentDocument.documentElement?.offsetHeight ?? 0,
	iframe.contentDocument.body?.offsetHeight ?? 0,
	100
);
iframe.style.height = `${ height }px`;
```

(with your nice comment still in there obviously)

Reply:
Oh wait I'm totally misreading this. Blame the afternoon slump. You're good 👍